src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,8 +5,7 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
Please ensure that the 'Truck' class has a no-args constructor to satisfy the checklist requirement.